.github/workflows/codeql-analysis.yml

```diff
@@ -37,15 +37,15 @@ jobs:
         # Learn more about CodeQL language support at https://git.io/codeql-language-support
 
     steps:
-    - uses: step-security/harden-runner@f086349bfa2bd1361f7909c78558e816508cdc10 # v2.8.0
+    - uses: step-security/harden-runner@c6295a65d1254861815972266d5933fd6e532bdf # v2.11.1
       with:
         egress-policy: audit
     - name: Checkout repository
-      uses: actions/checkout@629c2de402a417ea7690ca6ce3f33229e27606a5
```

Comment:
This is a really odd one. https://github.com/actions/checkout/commit/629c2de402a417ea7690ca6ce3f33229e27606a5 doesn't resolve. But the workflow is able to use this sha. It appears to be a sha that references a replaced v2 tag:

```
Tagger: eric sciple <ericsciple@users.noreply.github.com>
Date:   Tue Nov 2 12:49:38 2021 -0500

Update v2 tag

commit ec3a7ce113134d7a93b817d10a8272cb61118579
```
https://github.com/actions/checkout/commit/ec3a7ce113134d7a93b817d10a8272cb61118579